upgrade/upgrade-scripts/upgrade-container

```diff
@@ -429,14 +429,36 @@ function migrate_file() {
 		cp "$path" "/var/lib/machines/${CONTAINER}${path}" ||
 			die "'cp $path' failed for '$CONTAINER'"
 	else
-		if [[ -f "/var/lib/machines/${CONTAINER}${path}" ]]; then
-			# If the target file doesn't exist in the host then
-			# we delete the file. This case could happen if the host
-			# configuration doesn't exist but a fresh install placed
-			# package default configuration.
-			rm "/var/lib/machines/${CONTAINER}${path}" ||
-				die "failed to remove '$path' in '$CONTAINER'"
-		fi
+		#
+		# If the target file doesn't exist in the host then we
+		# delete the file. This case could happen if the host
+		# configuration doesn't exist but a fresh install placed
+		# package default configuration.
+		#
+		rm -f "/var/lib/machines/${CONTAINER}${path}" ||
```

Comment:
NIT: `rm -f` never fails so you don't need the `die`

Reply:
It won't fail if the file doesn't exist, but presumably the command execution could potentially fail for other reasons (signal, etc.)?

Reply:
> It won't fail if the file doesn't exist, but presumably the command execution could potentially fail for other reasons (signal, etc.)?

That's right, it would fail if the fail exists and the removal fails, which is something we'd want to catch.